lib/widgets/auth/email_auth_options_page.dart

```diff
@@ -19,62 +19,55 @@ class EmailAuthOptionsPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        resizeToAvoidBottomInset: false,
-        appBar: AppBar(
-          backgroundColor: Colors.white,
-          iconTheme: IconThemeData(
-            color: Colors.black,
-          ),
+      appBar: AppBar(
+        backgroundColor: Colors.white,
+        iconTheme: IconThemeData(
+          color: Colors.black,
         ),
-        body: StoreConnector<AppState, VmEmailAuthOptionsPage>(
-          distinct: true,
-          converter: (store) => store.state.emailAuthOptionsPage,
-          builder: (context, vm) {
-            if (vm.step != AuthStep.waitingForInput) {
-              return Center(child: CircularProgressIndicator());
-            }
-            return Center(
```

Comment:
i removed uneccessary Columns, Containers. These were to help with layout in old redundant 'otherAuthOptionsPage'